src/ipc/ipc4/handler.c

```diff
@@ -1325,7 +1325,11 @@ struct ipc_cmd_hdr *ipc_prepare_to_send(const struct ipc_msg *msg)
 void ipc_boot_complete_msg(struct ipc_cmd_hdr *header, uint32_t data)
 {
 	header->pri = SOF_IPC4_FW_READY;
-	header->ext = 0;
```

Comment:
why remove this? why not keep it and only have the if below?

Reply:
sure, I get your idea. Thanks! update.